microsoft-edge/progressive-web-apps/samples/index.md

```diff
@@ -113,21 +81,33 @@ Application Title Meta Tag uses the following features:
 
 <!-- ====================================================================== -->
 ## Temperature converter
+<!-- "MicrosoftEdge / Demos" repo; repo dir: __ -->
 
-A simple PWA demo app that converts temperatures.  Used for [Get started with PWAs](./how-to/index.md).<!-- todo: change from [Get started with PWAs] to [Temperature convertor sample] after merge https://github.com/MicrosoftDocs/edge-developer/pull/3476 -->
+A simple PWA demo app that converts temperatures.  See [Temperature converter sample](./temperature-converter.md).
 
-[App](https://microsoftedge.github.io/Demos/pwa-getting-started/), [source code and Readme file](https://github.com/MicrosoftEdge/Demos/tree/main/pwa-getting-started).
+* [App](https://microsoftedge.github.io/Demos/pwa-getting-started/)
+* [/pwa-getting-started/](https://github.com/MicrosoftEdge/Demos/tree/main/pwa-getting-started/) - source code directory.
+   * [Readme file](https://github.com/MicrosoftEdge/Demos/blob/main/pwa-getting-started/README.md)
 
 
 <!-- ====================================================================== -->
 ## PWA installer
+<!-- 
+## PWA store
+-->
+<!-- "MicrosoftEdge / Demos" repo; repo dir: __ -->
 
 A PWA that uses the Web Install API to install other PWAs.  Also uses CSS Masonry.
 
-[App](https://microsoftedge.github.io/Demos/pwa-pwastore/), [source code and Readme file](https://github.com/MicrosoftEdge/Demos/tree/main/pwa-pwastore).
-<!-- todo: new:
-[App](https://microsoftedge.github.io/Demos/pwa-installer/), [source code and Readme file](https://github.com/MicrosoftEdge/Demos/tree/main/pwa-installer/).
--->
+Original version:<!-- todo: which should the user use?  why 2 dirs? -->
```

Comment:
live article
https://learn.microsoft.com/microsoft-edge/progressive-web-apps/demo-pwas#pwa-installer
links to /pwastore/ only, not to /pwa-installer/ - which sample(s) should it link to?  

The article should explicitly address this and discuss relationship between the two samples/dirs.

Maybe nowhere mention pwastore, but in its Readme, explain relation between the two samples/dirs.